packages/react/src/components/UIShell/HeaderContainer.js

```diff
@@ -10,10 +14,13 @@ const HeaderContainer = ({ isSideNavExpanded, render: Children }) => {
     setIsSideNavExpandedState(!isSideNavExpandedState);
   }, [isSideNavExpandedState, setIsSideNavExpandedState]);
 
+  headerNavRef = React.createRef();
```

Comment:
does the `ref` need to persist between renders? we can't use `createRef` in a function component right?